KCSs/kcs-2.md

```diff
@@ -16,13 +16,21 @@ NFTs using this standard may include additional utility and functionality beyond
 
 ## Why
 
-By mimicing the Ethereum NFT collection standard this makes it easy to onboard NFT projects who may have already launched collections on other chains. Many tools and resources already available for working with NFT collections will already be compatible here and easy to implement on Koinos.
+By mimicking the Ethereum NFT collection standard this makes it easy to onboard NFT projects who may have already launched collections on other chains. Many tools and resources already available for working with NFT collections will already be compatible here and easy to implement on Koinos.
 
 Further, by setting forth a standard for NFT collections developers can be sure that NFT marketplaces (including [Kollection](https://kollection.app)) will be able to aggregate and display these NFT collections.
 
+## Why the concept of "owner"?
+
+On Koinos contracts can be either upgradeable or immutable. A contract address is also a standard wallet address. These are differences that make our implementation slightly different from ERC-721.
+
+By default, the wallet that can make changes to a contract is the address that uploaded the contract. If a contract is set to immutable (by blocking authorities) then the wallet used to upload the contract can no longer make changes. In our implementation of KCS-2 there is the concept of an "owner" address and the ability to transfer that ownership. That means that a collection owner could transfer the ownership of the collection to another individual or organization even if the contract is set to be immutable. This is a concept that is built-in to Ethereum for contracts, but, is "unnecessary" to be default behavior on Koinos because standard wallet addresses can be contract addresses.
```

Comment:
could you clarify this part `This is a concept that is built-in to Ethereum for contracts` ? I think you refer to the fact that all contracts are immutable by default, right? ... this paragraph can imply that you mean that the transfer of the ownership is built-in to Ethereum, which is not true.

Reply:
Yes you're right - `owner` is implemented in ERC-721 as part of that contract standard but not on Ethereum itself. I pushed an update to clarify and remove that text.